src/main/java/mate/academy/dao/ConnectionDB.java

```diff
@@ -0,0 +1,27 @@
+package mate.academy.dao;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+import mate.academy.exception.DataProcessingException;
+
+public class ConnectionDB {
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/test";
+    private static final String JDBC_DRIVER = "com.mysql.cj.jdbc.Driver";
+    private static final Properties DB_PROPERTIES = new Properties();
+
+    static {
+        DB_PROPERTIES.put("user", "root");
+        DB_PROPERTIES.put("password", "qwerty123456");
+        try {
+            Class.forName(JDBC_DRIVER);
+        } catch (ClassNotFoundException e) {
+            throw new DataProcessingException("Can't load mysql driver ", e);
```

Comment:
I'm not sure that `DataProcessingException` which u use in your daos is a good candidate for this place. You can use `RuntimeException` here for example.